test/functional/ui/output_spec.lua

```diff
@@ -22,6 +22,7 @@ describe("shell command :!", function()
 
   after_each(function()
     screen:detach()
+    session.command('call jobstop(b:terminal_job_id)')
   end)
 
   it("displays output even without LF/EOF. #4646 #4569 #3772", function()
```

Comment:
I could swear I checked every test in #5197 at least twice... Anyway thanks for pointing it out!